content/docs/command-reference/init.md

```diff
@@ -32,67 +32,59 @@ that are hidden from the user. This directory is automatically staged with
 
 `--subdir` must be provided to initialize DVC in a subdirectory of a Git
 repository. DVC still expects to find the Git repository (will check all
-directories up to the root to find `.git`). This options does not affect any
-config files, `.dvc` directory is created the same way as in the default mode.
-This way multiple DVC projects (including nested ones) could be initialized in a
-single Git repository providing isolation and granular project management.
+directories up to the system root to find `.git/`). This options does not affect
+any config files, `.dvc/` directory is created the same way as in the default
+mode. This way multiple <abbr>DVC projects</abbr> can be initialized in a single
+Git repository, providing isolation between projects.
 
 #### When is this useful?
 
-`--subdir` is mostly used in the scenario of a
-[monorepo](https://en.wikipedia.org/wiki/Monorepo), but also can be used in
-other workflows when such isolation and/or advanced granularity is needed.
+This option is mostly useful in the scenario of a
+[monorepo](https://en.wikipedia.org/wiki/Monorepo) (Git repository split into
+several project directories), but can also be used with other patterns when such
+isolation is needed. `dvc init --subdir` mitigates the issues of initializing
+DVC in the Git repository root:
 
-Let's imagine we have an existing Git repository that is split into sub-projects
-(monorepo). In this case `dvc init --subdir` can be run in one or many
-sub-projects to mitigate the issues of initializing in the Git repository root:
+- Repository maintainers might not allow a top level `.dvc/` directory,
+  especially if DVC is being used by several sub-projects (monorepo).
 
-- Repository maintainers might not allow extra `.dvc` top level directory,
-  especially if DVC is being used by a small number of sub-projects.
+- DVC config file, cache directory,
+  [etc.](/doc/user-guide/dvc-files-and-directories) are shared across different
+  sub-projects. This makes it difficult to use different DVC settings,
+  [remote storage](/doc/command-reference/remote) locations, etc.
 
-- Not enough isolation/granularity - DVC config, cache, and other files are
-  shared across different sub-projects. Means that it's not easy to use
-  different remote storages, for example, for different sub-projects, etc.
-
-- Not enough isolation/granularity - commands like `dvc pull`, `dvc checkout`,
-  and others analyze the whole repository to look for `dvc.yaml` or `.dvc` files
-  to download files and directories, to reproduce <abbr>pipelines</abbr>, etc.
-  It can be expensive in the large repositories with a lot of projects.
-
-- Not enough isolation/granularity - commands like `dvc metrics diff`, `dvc dag`
-  and others by default dump all the metrics, all the pipelines, etc.
+- Many DVC commands can explore the whole <abbr>DVC repository</abbr> to find
```

Comment:
I would prefer to keep it more explicit as it was before. Even if we merge two points.

Reply:
OK. You mean to add "Not enough isolation" to the beginning of every bullet? It seemed repetitive to me especially since "isolation" is already mentioned several times in preceding paragraphs but as you prefer.

Reply:
isolation +/- granularity is minor thing. I guess two paragraphs with specific commands to give some sense, etc.

Reply:
OK, added back a couple specific command examples in 221ed757.